### ISSUE TITLE
A question about the 'flow.flo' file in images.

### ISSUE THREAD

**Issue by Fairydetail** [sniklaus/softmax-splatting#20]
Hello, Simon. Thank you for your code sharing.
There is a file named 'flow.flo' in the project. And is it computed by the finetuned pwcnet model?
If so, is it convenient for you to release the finetuned pwcnet model?

**Comment by sniklaus** [comment]
It is computed using a PWC-Net that was fine-tuned within our interpolation pipeline (and not the initial/raw PWC-Net that was only trained on FlyingChairs). I am afraid that I am not allowed to share any parts of the trained pipeline, my apologies.
